### PR TITLE
Buttons are hidden until code is added

### PR DESCRIPTION
**Pull request by akanshgulati**
###### Updated the disabling of buttons if no content is there.

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] Improvement
    - [ ] Enhancement: **`{JSON Validator}: {Not Validate JSON is disabled untill content is not there}`**

### CODE REVIEW

**Comment by edgesince84** [comment]
thanks for this!  I'm going to loop in @sahildua2305 👍

**Comment by sahildua2305** [comment]
Thanks @akanshgulati! I'll check this and let you know.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@ DDG-*
 .build
 *.sw*
 *~
-share/goodie/meta/metadata.json
+share/goodie/meta/metadata.json
```
Comment (sahildua2305) [inline .gitignore]: Please revert the changes to this file completely. (End line is missing)

Comment (akanshgulati) [inline .gitignore]: I guess, it is achieved now.

**Comment by akanshgulati** [comment]
@sahildua2305  any update on this PR?

**Comment by sahildua2305** [comment]
@akanshgulati Thanks you for your inputs on this. I have pushed a better (read as more neat) solution to disable validate button when contents are empty. Please have a look and give your valuable suggestions, if any!

**Comment by akanshgulati** [comment]
Looks more user friendly. Actually I also did that but forgot to push it as I started discussing on css of disabling button.

**Comment by sahildua2305** [comment]
No problem. Please keep contributing. Don't get disheartened. :)
